doc/UsersGuide/source/BuildingAndRunning.rst

```diff
@@ -154,19 +150,56 @@ to build the prerequisite libraries, there is a script in the ``NCEPLIBS-ufs-v2.
 
 Of course, you can also set the values of these variables yourself if you know where the paths are on your system.
 
---------------------------------------------
-Setting the CCPP_SUITES environment variable
---------------------------------------------
+-------------------------------------------------------------
+Setting the CMAKE_FLAGS and CCPP_SUITES environment variables
+-------------------------------------------------------------
+
+You need to use the ``CMAKE_FLAGS`` environment variable to specify which application to build.
+In order to have one or more CCPP physics suites available at runtime, you also need to select those suites at
+build time by setting the ``CCPP_SUITES`` environment variable. Multiple suites can be set. Following
+examples are for the bash shell.
+
+For the ufs-weather-model ATM app (standalone ATM):
+
+.. code-block:: console
+
+    export CMAKE_FLAGS="-DAPP=ATM"
+    export CCPP_SUITES="FV3_GFS_v16"
 
-In order to have one or more CCPP physics suites available at runtime, you need to select those suites at
-build time by setting the ``CCPP_SUITES`` environment variable. Multiple suites can be set, as shown below
-in an example for the bash shell:
+For the ufs-weather-model ATM app (standalone ATM) in 32 bit:
 
 .. code-block:: console
 
-    export CCPP_SUITES="FV3_GFS_v15p2,FV3_GFS_v16beta"
+    export CMAKE_FLAGS="-DAPP=ATM -D32BIT=ON"
+    export CCPP_SUITES="FV3_GFS_v16"
+
+For the ufs-weather-model ATMW app (standalone ATM with wave):
+
+.. code-block:: console
+
+    export CMAKE_FLAGS="-DAPP=ATMW"
+    export CCPP_SUITES="FV3_GFS_v16"
+
+For the ufs-weather-model S2S app (atm/ice/ocean):
+
+.. code-block:: console
+
+    export CMAKE_FLAGS="-DAPP=S2S"
+    export CCPP_SUITES="FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled,FV3_GFS_v16_coupled,FV3_GFS_v16_couplednsst"
+
+For the ufs-weather-model S2S app (atm/ice/ocean) with debugging flags turned on, with verbose build messages:
```

Comment:
BUILD_VERBOSE=1 must be exported to get verbose build messages

Reply:
Will include that.